supportedBackends/sge/src/main/scala/cromwell/backend/impl/sge/SgeInitializationActor.scala

```diff
@@ -20,8 +20,8 @@ object SgeInitializationActor {
 }
 
 class SgeInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
-                             override val calls: Seq[Call],
-                             override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+                                      override val calls: Seq[Call],
+                                      override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
```

Comment:
Better to make whitesnake comments

![url](https://cloud.githubusercontent.com/assets/961771/16019133/7421a59a-3175-11e6-85cf-30aefc9b092b.jpg)